package.json

```diff
@@ -14,7 +14,7 @@
     "lint:js:fix": "eslint src --fix",
     "patch:hosts": "fec patch-etc-hosts",
     "start": "HOT=true fec dev",
-    "test": "TZ=UTC jest --verbose --no-cache",
+    "test": "TZ=UTC jest --verbose --no-cache --runInBand",
```

Comment:
FWIW without `--runInBand` the tests fail in the Konflux pipeline consistently. We will have to follow up on this.